pkg/controllers/management/auth/globalroles/globalrolebinding_handler.go

```diff
@@ -593,4 +606,10 @@ func isCRTBValid(crtb *v3.ClusterRoleTemplateBinding, cluster *v3.Cluster, bindi
 		crtb.UserName == binding.UserName &&
 		crtb.GroupPrincipalName == binding.GroupPrincipalName &&
 		crtb.DeletionTimestamp == nil
+// IsClusterUnavailable returns true if the cluster is not reachable, false otherwise.
```

Comment:
Is there a missing closing brace for the above function, or am I reading this wrong?

Reply:
Accidental from the last set of merge conflicts. It's fixed and I made sure the repo builds successfully without errors